client/src/components/BoulderCard.jsx

```diff
@@ -77,11 +70,14 @@ const BoulderCard = ({ boulder, detailedMode }) => {
 
 export default BoulderCard;
 
-const WrapperShort = styled.li`
+const WrapperShort = styled(Link)`
```

Comment:
`WrapperShort` should be the `listitem`, as you are rendering your `BoulderCard` in a `ul`. The link should be inside the `listitem`.